prusti-interface/src/environment/traits/mod.rs

```diff
@@ -1,5 +1,4 @@
 mod rustc_codegen;
 mod rustc_instance;
 
-pub(super) use rustc_instance::resolve_instance;
-pub(super) use rustc_codegen::codegen_fulfill_obligation;
```

Comment:
Did you forget to delete `codegen_fulfill_obligation`?

Reply:
No, it is still used from withing `resolve_instance`… I might delete the (one) usage of that method eventually, but not in this PR.